ignite/metrics/accuracy.py

```diff
@@ -14,11 +14,48 @@ class Accuracy(Metric):
     - `y_pred` must be in the following shape (batch_size, num_categories, ...) or (batch_size, ...)
     - `y` must be in the following shape (batch_size, ...)
     """
+    def __init__(self, is_multilabel=False, threshold_function=lambda x: torch.round(x), output_transform=lambda x: x):
+        super(Accuracy, self).__init__(output_transform)
+        self._threshold = threshold_function
+        if is_multilabel:
+            self.update = self._update_multilabel
+        else:
+            self.update = self._update_multiclass
```

Comment:
Here we need to raise error if user specifies `self._threshold`. So maybe better to set `threshold_function=None` by default and then setup as `torch.round(x)` if needed in multi-label case.

Reply:
Will incorporate. added in latest commit.